.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"

```

```suggestion
      interval: "weekly"

```
@amandesai01 This could be daily, weekly and monthly. Please let me know your thought